### PR TITLE
Kotlin in Datalore

### PR DESCRIPTION
**Pull request by AlexBartholomew7**
Suggest adding kotlin_in_datalore.png to https://github.com/JetBrains/kotlin-web-site/tree/master/docs/images/data-science
and using the commented-out

### CODE REVIEW

File: docs/topics/data-science-overview.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ support to Jupyter Notebook.
...
 instructions, documentation, and examples.
 
+### Kotlin Notebooks in Datalore
+
+With Datalore, you can start using Kotlin in the browser straight out of the box, no installation required. You can also collaborate on Kotlin notebooks in real time, get smart coding assistance when writing code, and share results as interactive or static reports. Check out a [sample report](https://datalore.jetbrains.com/view/report/9YLrg20eesVX2cQu1FKLiZ).
+
+<!--<img src="kotlin_in_datalore.png" alt="Kotlin in Datalore" width="800"/>-->
+![kotlin_in_datalore](https://user-images.githubusercontent.com/73881741/181792090-9c9e0d75-1c2c-4e53-8e56-349b8bfe9921.png)
+     
+Use Kotlin with a free Datalore Community account. [Register here].
```
Comment (koshachy) [inline docs/topics/data-science-overview.md:36]: Could you provide a link here?

File: docs/topics/data-science-overview.md [hunk 2]
```diff
@@ -26,6 +26,17 @@ support to Jupyter Notebook.
 Check out Kotlin kernel's [GitHub repo](https://github.com/Kotlin/kotlin-jupyter) for installation 
 instructions, documentation, and examples.
 
+### Kotlin Notebooks in Datalore
+
+With Datalore, you can start using Kotlin in the browser straight out of the box, no installation required. You can also collaborate on Kotlin notebooks in real time, get smart coding assistance when writing code, and share results as interactive or static reports. Check out a [sample report](https://datalore.jetbrains.com/view/report/9YLrg20eesVX2cQu1FKLiZ).
```
Comment (koshachy) [inline docs/topics/data-science-overview.md:31]:
```suggestion
With Datalore, you can use Kotlin in the browser straight out of the box, no installation required. You can also collaborate on Kotlin notebooks in real time, get smart coding assistance when writing code, and share results as interactive or static reports. Check out a [sample report](https://datalore.jetbrains.com/view/report/9YLrg20eesVX2cQu1FKLiZ).
```

**Review by koshachy — CHANGES_REQUESTED** [review]
Thanks for the contribution.
Could you please take a look at the comments?

Feel free to contact me if you have any questions.

File: docs/topics/data-science-overview.md (hunk 1 above)
Comment (AlexBartholomew7) [reply docs/topics/data-science-overview.md:36]: http://jetbrains.com/datalore

File: docs/topics/data-science-overview.md [hunk 3]
```diff
@@ -26,6 +26,17 @@ support to Jupyter Notebook.
...
+
+<!--<img src="kotlin_in_datalore.png" alt="Kotlin in Datalore" width="800"/>-->
+![kotlin_in_datalore](https://user-images.githubusercontent.com/73881741/181792090-9c9e0d75-1c2c-4e53-8e56-349b8bfe9921.png)
```
[the remark this answers is not in the document]
Comment (AlexBartholomew7) [reply docs/topics/data-science-overview.md:34]: I cannot. I don't have access. 'Uploads are disabled'

[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/data-science-overview.md]: Oh, I see. So let's merge this one and then I'll update the pic in the separate PR.

**Review by koshachy — APPROVED** [review]
I'll upload and update the pic in the separate PR.

**Comment by koshachy** [comment]
https://github.com/JetBrains/kotlin-web-site/pull/3099

**Pull request merged**